src/terrain/path/tile_iterator.cpp

```diff
@@ -32,11 +32,16 @@ namespace terrain {
 
 namespace path {
 
+/* We have 26 offsets because 3^3-1 = 26
+   The total number of adjacent cubes is the number cubes in a three by three
+   area minus one for the central cube.
+ */
```

Comment:
Ah so it's a lookup table. Got it
```suggestion
/* We have 26 offsets because 3^3-1 = 26
 * The total number of adjacent cubes is the number cubes in a three by three
 * area minus one for the central cube.
 */
```